pandas/io/parsers/base_parser.py

```diff
@@ -1119,19 +1119,13 @@ def converter(*date_cols):
         if date_parser is None:
             strs = parsing.concat_date_cols(date_cols)
 
-            try:
-                return tools.to_datetime(
-                    ensure_object(strs),
-                    utc=False,
-                    dayfirst=dayfirst,
-                    errors="ignore",
-                    cache=cache_dates,
-                ).to_numpy()
-
-            except ValueError:
```

Comment:
does this induce an api change?

Reply:
shouldn't do, as `to_datetime` would already fall back to `du_parse` if not `format` is provided and if it can't infer the format